endpoints-support.md

```diff
@@ -182,8 +182,14 @@
 |<sub>/rest/switch-types</sub>                                                            | GET      | :white_check_mark:   | :white_check_mark:   | :heavy_multiplication_x:   |
 |<sub>/rest/switch-types/{id}</sub>                                                       | GET      | :white_check_mark:   | :white_check_mark:   | :heavy_multiplication_x:   |
 |     **Storage Volume Attachments**                                                                                                                                   |
-|<sub>/rest/storage-volume-attachments</sub>                                              | GET      | :heavy_multiplication_x:  | :heavy_multiplication_x: | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | 
-|<sub>/rest/storage-volume-attachments/{id}</sub>                                         | GET      | :heavy_multiplication_x:  | :heavy_multiplication_x: | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/storage-volume-attachments</sub>                                              | GET      | :heavy_multiplication_x:  | :heavy_multiplication_x: | :heavy_multiplication_x:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | 
```

Comment:
We are following alphabetical order for the resources, Can you please place the Switch Types below storage volume templates.

Reply:
Done.